android/src/main/java/com/reactnativestripesdk/pushprovisioning/PushProvisioningProxy.kt

```diff
@@ -0,0 +1,109 @@
+package com.reactnativestripesdk
+
+import android.app.Activity
+import android.app.Activity.RESULT_OK
+import android.content.Intent
+import android.util.Log
+import com.facebook.react.bridge.*
+import com.reactnativestripesdk.pushprovisioning.AddToWalletButtonView
+import com.reactnativestripesdk.pushprovisioning.EphemeralKeyProvider
+import com.reactnativestripesdk.pushprovisioning.TapAndPayProxy
+import com.stripe.android.pushProvisioning.PushProvisioningActivity
+import com.stripe.android.pushProvisioning.PushProvisioningActivityStarter
+
+
+object PushProvisioningProxy {
```

Comment:
`com.stripe.android.pushProvisioning` is an optional dependency, so we proxy to it from this object. If it's not found, we send an error event to the onComplete` hook, but we don't crash. Basically- all methods will be a no-op without that dependency.